website/README.md

```diff
@@ -554,6 +554,40 @@ extensions for your code editor. The
 [`.prettierrc`](https://github.com/explosion/spaCy/tree/master/website/.prettierrc)
 file in the root defines the settings used in this codebase.
 
+## Building & Developing the Site with Docker {#docker}
```

Comment:
Nitpick: I wouldn't use capitals across the title, most other sections in this readme only capitalize the first letter
```suggestion
## Building & developing the site with Docker {#docker}
```